numba/tests/npyufunc/test_vectorize_decor.py

```diff
@@ -28,88 +28,82 @@ def vector_add(a, b):
     return a + b
 
 
-class BaseVectorizeDecor(unittest.TestCase):
-    def _run_and_compare(self, numba_func, numpy_func):
-        A = np.arange(100, dtype=np.float64)
-        result = numba_func(A)
-        gold = numpy_func(A)
-        np.testing.assert_allclose(result, gold)
-
-    def _test_template_1(self, target):
-        numba_sinc = vectorize(['float64(float64)', 'float32(float32)'],
-                               target=target)(sinc)
-        numpy_sinc = np.vectorize(sinc)
-        self._run_and_compare(numba_sinc, numpy_sinc)
-
-    def _test_template_2(self, target):
-        numba_sinc = vectorize([float64(float64), float32(float32)],
-                               target=target)(sinc)
-        numpy_sinc = np.vectorize(sinc)
-        self._run_and_compare(numba_sinc, numpy_sinc)
-
-    def _test_template_3(self, target):
-        numba_scaled_sinc = vectorize(['float64(float64, uint32)'],
-                                      target=target)(scaled_sinc)
-        numpy_scaled_sinc = np.vectorize(scaled_sinc)
-        A = np.arange(100, dtype=np.float64)
-        scale = np.uint32(3)
-        result = numba_scaled_sinc(A, scale)
-        gold = numpy_scaled_sinc(A, scale)
-        np.testing.assert_allclose(result, gold, atol=1e-8)
-
-    def _test_template_4(self, target):
-        sig = [int32(int32, int32),
-               uint32(uint32, uint32),
-               float32(float32, float32),
-               float64(float64, float64)]
-        basic_ufunc = vectorize(sig, target=target)(vector_add)
-        np_ufunc = np.add
-
-        def test(ty):
-            data = np.linspace(0., 100., 500).astype(ty)
-            result = basic_ufunc(data, data)
-            gold = np_ufunc(data, data)
-            np.testing.assert_allclose(gold, result)
+class BaseVectorizeDecor(object):
 
-        test(np.double)
-        test(np.float32)
-        test(np.int32)
-        test(np.uint32)
+    __test__ = False
```

Comment:
...actually, adding the attribute in the base class means every subclass has to remember to override it otherwise they won't be run. Since it's not necessary in this case perhaps its best to simply remove it